Doc/using/venv-create.inc

```diff
@@ -61,12 +61,16 @@ The command, if run with ``-h``, will show the available options::
                             environment (pip is bootstrapped by default)
       --prompt PROMPT       Provides an alternative prompt prefix for this
                             environment.
-      --upgrade-deps        Upgrade core dependencies: pip setuptools to the
+      --upgrade-deps        Upgrade core dependencies: pip to the
```

Comment:
```suggestion
      --upgrade-deps        Upgrade core dependencies (pip) to the
```

Reply:
Done, and updated the code that this is supposed to be equivalent to as well.